frontend/src/Components/Customers/View.js

```diff
@@ -0,0 +1,50 @@
+import React, { useEffect, useState } from 'react';
+import axios from 'axios';
+import { Row } from './Row';
+
+export function View(props) {
+    const [customers, setCustomers] = useState([])
+    const [selectedRowIndex, setSelectedRowIndex] = useState(0)
+
+    useEffect(() => { axios.get('/customers').then(cust => { setCustomers(cust.data) }) }, [])
+
+    function onClick(index) {
+        setSelectedRowIndex(index)
+    }
+
+    function DisplayCustomers(searchText) {
+        let customerList = customers
```

Comment:
As fi folosit direct variabila ```customers```, nu as fi reatribuit-o in customerList

Reply:
Done.